src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/Organization.scala

```diff
@@ -1,4 +1,3 @@
 package ch.epfl.bluebrain.nexus.admin.organizations
-import java.util.UUID
 
-final case class Organization(id: UUID, label: String, description: String)
+final case class Organization(label: String, description: String)
```

Comment:
Should description be optional?

Reply:
I would say no, it will probably make it nicer to display on the frontend if it's mandatory.